src/client/components/draft/paragraph/utils/__tests__/utils.test.js

```diff
@@ -8,9 +8,9 @@ import {
   styleNamesFromMap,
   styleNodesFromMap
 } from '../utils'
-const SelectionUtils = require('../../../../rich_text/utils/text_selection.js')
+const SelectionUtils = require('../../../../rich_text/utils/text_selection')
```

Comment:
Can use absolute paths: https://github.com/artsy/positron/blob/master/.babelrc#L20

Reply:
The absolute paths seem to be broken in typescript